### ISSUE TITLE
Investigate false failures in daily accessibility scan

### ISSUE THREAD

**Issue by ddzz** [department-of-veterans-affairs/va.gov-team#35837]
## Issue Description
Recently the daily accessibility scan reported failures for accessibility issues that had already been fixed and merged to the `master` branch. This task is to determine why the daily scan is not picking up the latest changes and lagging behind what's on the `master` branch..

---
## Tasks
- [ ] Determine root cause of false failures in daily accessibility scan.

## Acceptance Criteria
- [ ] Daily accessibility scan is up-to-date with the `master` branch.

**Comment by pjhill** [comment]
Explored possibility of using dev build. That first attempt at a PR didn't work out. PR linked above.

**Comment by pjhill** [comment]
* Discussion with Theo
* Working locally
* Working on getting it up and running in GHA

**Comment by ddzz** [comment]
The Release Tools team mentioned we may want to keep the build as-is, so I'm pausing any more work on this until I get more details on why we should use production assets. My suspicion is that the deployed assets are built differently than the assets from the `master` commit.

**Comment by ddzz** [comment]
The PR has been tested and approved, but it can't be merged due to issues with the self-hosted runners used in GitHub Actions. The Infrastructure team is working on a fix.

**Comment by ddzz** [comment]
The Infrasturcture team has fixed the issues with the self-hosted runners, and the PR has been merged.

**Comment by pjhill** [comment]
Thanks for investigating this!
